app/serializers/site_serializer.rb

```diff
@@ -22,4 +22,11 @@ class SiteSerializer < ActiveModel::Serializer
 
   has_many :categories
   has_many :links
+
+  has_many :recent_posts do
+    link :related do
+      api_site_posts_path(object.fqdn, page: {size: 5})
```

Comment:
As an endpoint to fetch recent posts, using `/posts` looks like natural (RESTful?) instead of exposing the endpoint `/recent_posts`,  such as `/posts` with query parameters (a sort field and a count), `/posts/recent` with query parameters (a count) and so forth. It just popped up in my mind.